projects/RabbitMQ.Client/client/impl/ISession.cs

```diff
@@ -73,8 +73,8 @@ internal interface ISession
         void Close(ShutdownEventArgs reason, bool notify);
         bool HandleFrame(in InboundFrame frame);
         void Notify();
-        void Transmit(in OutgoingCommand cmd);
-        void Transmit(IList<OutgoingCommand> cmds);
-
+        void Transmit<T>(in T cmd) where T : struct, IOutgoingCommand;
```

Comment:
remove in favor of the one below with default parameters?

Reply:
possible, but there are 2 benefits by not doing so
- You either have just cmd or cmd, header & body. Default parameters would enable the possibility of just leaving out the body. 
- Performance, as the less that needs to be passed, the better. Also Framing.SerializeToFrames is more performant due to the reduced parameters